server/itx/heartbeat.go

```diff
@@ -91,6 +92,13 @@ func (h *HeartbeatHandler) Log() {
 		height = 0
 	}
 
+	// Actpool metrics
+	numActs := h.s.actPool.GetSize()
+	capacity := h.s.actPool.GetCapacity()
+	numUnconfirmedActs := h.s.actPool.GetUnconfirmedActSize()
+	s := fmt.Sprintf("size: %d/%d, unconfirmed actions: %d",
```

Comment:
you don't need to combine and format the output here. You could logger them as separate metrics